src/schemas/author_schema.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+from typing import Optional
+
+from pydantic import BaseModel, Field, validator
+
+
+class AuthorSchema(BaseModel):
+
+    """
+    Pydantic models for Author model
+
+    """
+
+    user_id: Optional[int] = None
+    first_name: str = Field(title="First name of Author ")
+    last_name: str = Field(title="Last name of Author ")
+    birth_date: datetime = Field(title="Date of Birth For The Author")
+    death_date: datetime = Field(title="Date of Passing For The Author", default=None)
+
+    @validator("death_date")
+    def birth_date_greater_than_death_date(cls, v, values, **kwargs):
+        if "birth_date" in values and values["death_date"] >= values["birth_date"]:
+            raise ValueError("Death Date must be greater than Birth Date")
+
+
+class Config:
```

Comment:
These config classes are subclasses of schema classes. So they should be inside schema models not out side.